keanu-project/src/main/java/io/improbable/keanu/distributions/continuous/MultivariateGaussian.java

```diff
@@ -11,10 +11,11 @@
     private final DoubleTensor mu;
     private final DoubleTensor covariance;
 
-    public static ContinuousDistribution withParameters(DoubleTensor mu, DoubleTensor covariance) {
+    public static MultivariateGaussian withParameters(DoubleTensor mu, DoubleTensor covariance) {
         return new MultivariateGaussian(mu, covariance);
     }
     private MultivariateGaussian(DoubleTensor mu, DoubleTensor covariance) {
+        TensorShapeValidation.checkShapesCanBeMatrixMultiplied(covariance.getShape(), mu.getShape());
```

Comment:
this check was missing previously - there was a check in MultivariateGaussianVertex but no check here.